### PR TITLE
Fix untranslatable photo upload button's hover text

### PR DESCRIPTION
**Pull request by AllDragonsAreDead**
Fix https://github.com/diaspora/diaspora/issues/7427

### CODE REVIEW

File: app/views/publisher/_publisher.mobile.haml [hunk 1]
```diff
@@ -31,7 +31,7 @@
       %ul#photodropzone
     #fileInfo-publisher
 
-    #file-upload-publisher{title: t('shared.publisher.upload_photos'), class: 'btn btn-default'}
+    #file-upload-publisher{class: 'btn btn-default'}
```
Comment (diaspora-code-review) [inline app/views/publisher/_publisher.mobile.haml]: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

**Review by SuperTux88 — APPROVED** [review]

**Review by Flaburgan — APPROVED** [review]
The code looks OK, I tested this locally manually adding the french translation and it works well!

**Comment by SuperTux88** [comment]
Merged as afef279affdd5ed276db7d941daf1ec2a0fbe004

Thanks for your contribution! 👍

**Comment by Flaburgan** [comment]
Yay! Congratulation for this first merged pull request!

**Pull request closed without merging**